### PR TITLE
Fixed the issues identified while internal adoption testing

### PR DESCRIPTION
**Pull request by 4manasa**
Added visibility condition to attachment field.
Edit action not working issue fixed.
Percentage component not loading properly issue fixed.
Action buttons not displaying on modal dialog issue fixed

### CODE REVIEW

File: src/components/templates/CaseView/index.ts [hunk 1]
```diff
@@ -42,6 +42,7 @@ class CaseView extends BridgeBase {
 
   currentCaseID = '';
   bHasNewAttachments = false;
+  editAction: any;
```
Comment (niallriddell) [inline src/components/templates/CaseView/index.ts:45]: Use of any.

**Review by niallriddell — CHANGES_REQUESTED** [review]
Please use narrower types

File: src/components/templates/CaseView/index.ts (hunk 1 above)
Comment (4manasa) [reply src/components/templates/CaseView/index.ts]: Updated

File: src/components/templates/CaseView/index.ts [hunk 2]
```diff
@@ -18,6 +18,13 @@ interface CaseViewProps {
   header: string;
 }
 
+interface editActionType {
+  ID: string;
+  links: any;
```
Comment (niallriddell) [inline src/components/templates/CaseView/index.ts:23]: Use of any?  Can we find a narrower type?

**Review by niallriddell — CHANGES_REQUESTED** [review]
See if you can find a better type for links.

**Review by niallriddell — APPROVED** [review]
Happy to approve, but Object is only slightly narrower than any.  Not so bothered as links does not appear to be directly referenced.

**Review by niallriddell — COMMENTED** [review]
lgtm

**Pull request merged**